### PR TITLE
Join Habitat Slack from community page

### PR DESCRIPTION
**Pull request by ryankeairns**
Adds a second call-to-action for people to join the Slack Habitat group... aka Habichat.

Signed-off-by: Ryan Keairns <rkeairns@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @magwalk to be a potential reviewer

File: www/source/stylesheets/_home.scss [hunk 1]
```diff
@@ -62,12 +62,18 @@ $home-medium-breakpoint: rem-calc(880);
 }
 
 .hero--content {
-  margin: rem-calc(120) 0 rem-calc(400);
+  margin: rem-calc(60) 0 rem-calc(400);
```
Comment (ryankeairns) [inline www/source/stylesheets/_home.scss]: This undoes some extra padding we added for the announcement banner.

**Comment by magwalk** [comment]
@thesentinels r+

**Comment by juliandunn** [comment]
@thesentinels r+

**Comment by thesentinels** [comment]
:pushpin: Commit 1fe5a9b has been approved by `juliandunn`

**Comment by thesentinels** [comment]
:hourglass: Testing commit 1fe5a9b with merge 1ec0489...

**Comment by thesentinels** [comment]
:sunny: Test successful - [travis](https://travis-ci.com/habitat-sh/habitat/builds/26840341)

**Pull request merged**